RecoBTag/ImpactParameter/python/pfImpactParameterTagInfos_cfi.py

```diff
@@ -19,3 +19,7 @@
     candidates = cms.InputTag("particleFlow"),
     maxDeltaR = cms.double(0.4)
 )
+
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
+(pp_on_AA_2018 | pp_on_PbPb_run3).toModify(pfImpactParameterTagInfos, jets = "akCs4PFJets")
```

Comment:
is this enough to correctly run `pfImpactParameterTagInfos` for HI?
(similar question applies to the rest).

Reply:
No, it was not sufficient to modify pfImpactParamterTagInfos.  The soft lepton tag infos had to use the consistent jet collection to avoid crashes.

Reply:
I didn't mean the soft lepton taggers.
My question was if the switch of the jet for `pfImpactParameterTagInfos` expected to provide performance of `pfImpactParameterTagInfos` acceptable for physics in HI or if more significant changes are needed.

Reply:
The difference between impactParamterTagInfos and pfImpactParameterTagInfos is basically technical.  There should not be any significant difference in performance.   I verified this some time ago.  We will check this again before actually calculating b-tag discriminators based on the candidate tagInfos.  At the moment, they are not really used for anything in the HI workflow besides this validation package, AFAIK.